### ISSUE TITLE
loss = loss_func(input, prediction) should be loss = loss_func(target, prediction)

### ISSUE THREAD

**Issue by yezhengli-Mr9** [DanielWicz/trashanomaly#1]
`loss = loss_func(input, prediction)` should be `loss = loss_func(target, prediction)` in cell [0] `def train_step(...)` of [tutorial_notebook.ipynb](https://github.com/DanielWicz/trashanomaly/blob/master/tutorial_notebook.ipynb).
